src/Apps/W1/EDocument/App/app.json

```diff
@@ -0,0 +1,63 @@
+{
+  "id": "e1d97edc-c239-46b4-8d84-6368bdf67c8b",
+  "name": "E-Document Core",
+  "publisher": "Microsoft",
+  "brief": "The Dynamics 365 Business Central E-Documents module enables different models of electronic invoicing, available for additional localizations.",
+  "description": "Business Central's E-Documents module is the foundation layer for all e-invoicing standards covering most common processes, but it can be used for other electronic documents. The module is easily extendable with the country-based e-invoicing apps. The E-Documents app covers both sales and purchase processes and can have different lifecycles from standard invoices in Business Central.",
+  "version": "28.0.0.0",
+  "privacyStatement": "https://go.microsoft.com/fwlink/?LinkId=724009",
+  "EULA": "https://go.microsoft.com/fwlink/?linkid=2009120",
+  "help": "https://go.microsoft.com/fwlink/?linkid=2204541",
+  "url": "https://go.microsoft.com/fwlink/?LinkId=724011",
+  "logo": "ExtensionLogo.png",
+  "contextSensitiveHelpUrl": "https://learn.microsoft.com/en-us/dynamics365/business-central/",
+  "dependencies": [],
+  "internalsVisibleTo": [
+    {
+      "id": "e1d97edc-c239-46b4-8d84-6368bdf67c8c",
+      "name": "E-Document Core Tests",
+      "publisher": "Microsoft"
+    },
+    {
+      "id": "9bdb0f21-7daf-4ee2-bd73-6e0a2da02a18",
+      "name": "E-Document AI Tests",
+      "publisher": "Microsoft"
+    },
+    {
+      "id": "de0dddf3-9917-430d-8d20-6e7679a08500",
+      "name": "E-Document Core Demo Data",
+      "publisher": "Microsoft"
+    },
+    {
+      "id": "14aa1237-2f69-4c25-9a68-fa7d54e08613",
+      "name": "Payables Agent",
+      "publisher": "Microsoft"
+    },
+    {
+      "id": "967eceac-106e-4102-b76a-fe5dc3d799e5",
+      "name": "Payables Agent Tests",
+      "publisher": "Microsoft"
+    }
+  ],
+  "screenshots": [],
+  "platform": "28.0.0.0",
+  "idRanges": [
+    {
+      "from": 6100,
+      "to": 6195
+    }
+  ],
+  "resourceExposurePolicy": {
+    "allowDebugging": true,
+    "allowDownloadingSource": true,
+    "includeSourceInSymbolFile": true
+  },
+  "application": "28.0.0.0",
+  "target": "OnPrem",
```

Comment:
This should ideally be Cloud! You can get exceptions for certain things (such as Key Vault access).